transformer_engine/common/CMakeLists.txt

```diff
@@ -33,13 +39,30 @@ target_include_directories(transformer_engine PUBLIC
 # Configure dependencies
 target_link_libraries(transformer_engine PUBLIC
                       CUDA::cublas
+                      CUDA::cuda_driver
                       CUDA::cudart
+                      CUDA::nvrtc
                       CUDA::nvToolsExt
-		      CUDNN::cudnn)
+                      CUDNN::cudnn)
 target_include_directories(transformer_engine PRIVATE
                            ${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES})
```

Comment:
Since we are not creating pip wheels yet anyway, maybe we could use that information about CUDA header location from CMake?

Reply:
I've added the build-time header path to the list of search paths. It's low on the list in case we're cross-compiling.